dynostream-replicator.js

```diff
@@ -0,0 +1,70 @@
+var AWS = require('aws-sdk');
+var queue = require('queue-async');
+var _ = require('underscore');
+var streambot = require('streambot');
+
+module.exports = replicate;
+module.exports.streambot = streambot(replicate);
+
+function replicate(records, callback) {
+    var replicaConfig = {
+        region: process.env.ReplicaRegion,
+        endpoint: process.env.ReplicaEndpoint
+    };
+    /*
+    var replicaConfig = {
+        region: 'us-east-1',
+        endpoint: 'https://preview-dynamodb.us-east-1.amazonaws.com'
+    };
+    */
+
+    var replica = new AWS.DynamoDB(replicaConfig);
+
+    var recordsById = actionsPerId(records);
+
+    var q = queue();
+
+    _.forOwn(recordsById, function(records) {
+        q.defer(function(nextId) {
+            var serial = queue(1);
+
+            records.forEach(function(record) {
+                serial.defer(function(nextRecord) {
+                    console.log(record.EventID);
```

Comment:
Streambot provides a dedicated logger that writes logs to S3 (which can be ingested by other services). You should use that logger for any important logging, but also make sure that you're not sending excessive or duplicative log messages.